doc/news/changes/minor/20160214MathiasMentler

```diff
@@ -0,0 +1 @@
+Fixed: double contraction of two SymmetricTensors<..,VectorizedArray<T>> now
works. Introduced NumberType<T> with static member ::value to be called instead
of the constructor Number() in symmetric_tensor.h.
<br>
(Mathias Mentler, 2016/02/14)


```

Comment:
Oh wow, I guess this just isn't my day :) - Funny thing is I don't see that character neither in git nor in my editor (SublimeText).